src/bun.js/node/node_crypto_binding.zig

```diff
@@ -79,6 +108,7 @@ pub fn createNodeCryptoBindingZig(global: *JSC.JSGlobalObject) JSC.JSValue {
     crypto.put(global, bun.String.init("pbkdf2"), JSC.JSFunction.create(global, "pbkdf2", pbkdf2, 5, .{}));
```

Comment:
The inline capacity needs to be increased to 4

const crypto = JSC.JSValue.createEmptyObject(global, 4);